docs/user/secure-gateways.md

```diff
@@ -0,0 +1,214 @@
+# Secure Gateways
+
+This guide will help you get started using secure Gateways. The guide uses a self-signed CA, so it should be used for
+testing and demonstration purposes only.
+
+## Prerequisites
+
+- A Kubernetes cluster with `kubectl` context configured for the cluster.
+- OpenSSL to generate TLS assets.
+
+__Note:__ Envoy Gateway is tested against Kubernetes v1.24.
```

Comment:
Getting a little off-topic, but: Should we figure out a matrix of Kube versions that we want to test against?

Reply:
Slated for v0.3.0, [xref](https://github.com/envoyproxy/gateway/issues/493).